contracts/adapters/KycOnboarding.sol

```diff
@@ -168,6 +170,10 @@ contract KycOnboardingContract is AdapterGuard, Signatures {
         );
         bank.registerPotentialNewInternalToken(DaoHelper.UNITS);
         bank.registerPotentialNewToken(tokenAddr);
+
+        if (dao.getAddressConfiguration(DefaultTokenAddr) == address(0x0)) {
```

Comment:
Since `ETH_TOKEN` is already 0x0 address is there potential that if it gets configured first as the default that it will get overwritten as the default by the next token configured?